index.html

```diff
@@ -197,20 +209,28 @@ <h2>Skills</h2>
       </div>
     </section>
     <section class="contact">
-      <div class="heading">
-        <h2>
-          I'm always <br> interested in hearing <br>
-          about new projects, <br> so if you'd like to
-          <br> chat please get in touch.
-        </h2>
-      </div>
-      <div class="form">
-        <form action="https://formspree.io/f/meqdgvvo" method="post" id="contact-form">
-          <input type="text" placeholder="Full name" name="name" maxlength="30" required>
-          <input type="email" placeholder="Email address" name="email" required>
-          <textarea name="message" id="msg" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
-          <button type="submit" class="submit">Get in touch</button>
-        </form> 
+      <div class="row1">
+        <div class="heading">
+          <h2 class="contact-title">
+            <span class="mb-content">
+              I'm always <br> interested in hearing <br>
+              about new projects, <br> so if you'd like to
+              <br> chat please get in touch.
+            </span>
+            <span class="ds-content">
+              I'm always interested <br> in hearing about new <br> projects, so if you'd <br> like to chat 
+              please <br> get in touch.
+            </span>
+          </h2>
+        </div>
+        <div class="form">
+          <form action="https://formspree.io/f/meqdgvvo" method="post" id="contact-form">
+            <input type="text" placeholder="Full name" name="name" maxlength="30" required>
+            <input type="email" placeholder="Email address" name="email" required>
+            <textarea name="message" id="msg" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
+            <button type="submit" class="submit">Get in touch</button>
+          </form> 
+        </div>
       </div>
       <footer class="social-link">
         <hr class="line-2">
```

Comment:
Please adjust your footer line to correspond to the Figma design. It should be slightly thinner and with a 100% width.

Figma design:
![image](https://user-images.githubusercontent.com/42799579/193002939-ec494e99-2f79-4747-8dca-00374c1084e6.png)

Your design:
![image](https://user-images.githubusercontent.com/42799579/193002810-8016bd84-0b22-4629-b625-ab26ec5c9e1a.png)